docs/reference/project/schedules.mdx

```diff
@@ -1,48 +1,69 @@
+> Schedules are currently only supported on AWS and GCP. With support for other platforms coming soon.
+
 Configuring cron jobs directly on servers used to be quite common. However, Serverless environments where servers only run in response to requests/load make this more challenging. Directly configuring servers was also error prone, since the schedule configuration didn't employ version control and CI/CD processes.
 
```

Comment:
Would be good to add some comments around time zone challenges with scheduling. 

Looking at AWS and GCP documentation they talk about schedules being UTC based. This would be good to mention.
https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html
https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules

Reply:
I added a mention to time zones with the links near the bottom of page.